packages/binutils-oracle/2.27/version.desc

```diff
@@ -0,0 +1,13 @@
+# Since Oracle provides different branches for binutils
+# depending of the architecture:
+#
+#    ARM64: 'oracle/binutils/ol7-u9-aarch64'
+#    ARM32: 'oracle/binutils/ol7-u9-arm'
+#
+# branch name is adjusted in:
+#   samples/{aarch64,arm}*ol7u9-*
+#
+# using CT_BINUTILS_ORACLE_DEVEL_BRANCH
+#
+# This package will be "OBSOLETE" at EoS date for OL7: Jul 2014.
```

Comment:
I was also meaning this and the other packages should have `obsolete='yes'` to require people to select `CT_OBSOLETE` to get these older versions. The comment is mainly for the maintainers so we don't unintentionally remove these again until after the EoS date.

Reply:
> 2024

ohh no.. i'm a fool .. sorry for this.

Reply:
> I was also meaning this and the other packages should have `obsolete='yes'` to require people to select `CT_OBSOLETE` to get these older versions. The comment is mainly for the maintainers so we don't unintentionally remove these again until after the EoS date.

Ooh, ok understood, there is a new push now with `obsolete='yes'`  in version.desc
Thanks @cpackham !